docker/Dockerfile

```
@@ -0,0 +1,25 @@
+FROM moveit/moveit2:jazzy-release
```

I would like to be able to support specific (and multiple) ROS2 distros with this Dockerfile, so I would like to add an argument here to specify the distro.

```suggestion
ARG DISTRO=jazzy
FROM ros:${TAG}
ARG DISTRO
```

I think I would also prefer to build off of a base ROS2 image instead of the MoveIt image. `rosdep` should still install everything correctly and the build shouldn't take too much longer

---

Alternatively, we could have a single argument, say `${BASE_IMAGE}` that is provided by docker compose, and later during the `colcon build` step, we can first source `/opt/ros/*/setup.bash` without needing to know the distro. That way we could build off of base ROS images in CI and users could build off of `moveit` images using the `docker-compose` file if desired

---

I went for the DISTRO solution because it makes it clearer what we depend on as the base. I also don't expect to users to change the base very often. The build times differed very little.